Update driver support status (#11)
catie-aq/zephyr_omron-2smpb-02e: README.md

Omron 2SMPB 02E digital barometric pressure sensor Zephyr OS driver.

> [!NOTE]

> > This driver is now supported in the Zephyr OS mainline and is no longer maintained in this repository.

Reviewer: ```suggestion
> This driver has been contributed to the Zephyr OS mainline and is no longer maintained in this repository.
```
